src/generic/Router.ts

```diff
@@ -55,13 +55,34 @@ export class GenericWebhooksRouter {
         });
     }
 
+    private static xmlHandler(req: Request, res: Response, next: NextFunction) {
+        express.text({ type: ["*/xml", "+xml"] })(req, res, (err) => {
+            if (err) {
+                next(err);
+                return;
+            }
+            if (typeof req.body !== 'string') {
+                next();
+                return;
+            }
+            xml.parseStringPromise(req.body).then(xmlResult => {
+                req.body = xmlResult;
+                next();
+            }).catch(e => {
+                res.statusCode = 400;
+                next(e);
+            });
+        });
+    }
+
     public getRouter() {
         const router = Router();
         router.all(
             '/:hookId',
-            express.text({ type: 'text/*'}),
+            GenericWebhooksRouter.xmlHandler,
             express.urlencoded({ extended: false }),
```

Comment:
Is this for `x-www-form-urlencoded`?  And whatever it is, is it a distinct enough type to be documented in the "Payload formats" section?

Reply:
Good point. I think I missed it out.